HLTrigger/Configuration/python/HLT_75e33/modules/hgcalSoALayerClustersProducer_cfi.py

```diff
@@ -0,0 +1,13 @@
+import FWCore.ParameterSet.Config as cms
+from ..psets.hgcal_reco_constants_cfi import HGCAL_reco_constants as HGCAL_reco_constants
+
+hgCalSoALayerClustersProducer = cms.EDProducer("HGCalSoALayerClustersProducer@alpaka",
+    alpaka = cms.untracked.PSet(
+        backend = cms.untracked.string('cuda_async')
+    ),
+    hgcalRecHitsLayerClustersSoA = cms.InputTag("hgCalSoARecHitsLayerClustersProducer"),
+    hgcalRecHitsSoA = cms.InputTag("hgCalSoARecHitsProducer"),
+    mightGet = cms.optional.untracked.vstring,
```

Comment:
Here and in the other EDProducers, `mightGet` should be left out (unless explicitly set to a non-empty list)